ecomerce-app/src/component/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { FaHeart,FaShoppingCart } from "react-icons/fa";
+
+import  "./Navbar.module.css"
+
+export  function Navbar() {
```

Comment:
use ES6+ here too as you have used it everywhere in your other files. 

My suggestions are also the same as Rahul Shukla.

I really like how you used react-icons ... saw it for the first time.

Reply:
Thank you! I will change in next commit